src/modules/Channel/context/hooks/useGetChannel.ts

```diff
@@ -39,6 +40,7 @@ function useGetChannel(
           }
         })
         .catch((e) => {
+          eventHandlers?.request?.onFailed?.(e);
```

Comment:
I think actually the `logger.warn` should be done *before* calling `eventHandlers`. I think parsing logs would be more natural. Also applies to other files in this PR.

Reply:
Agreed.